examples/upbound-aws-provider/composite-resources/serverless-examples/sqs-lambda-s3/README.md

```diff
@@ -5,7 +5,7 @@ This example deploys the architecture depicted on the diagram. First, it applies
 
 ## Pre-requisites:
  - [Upbound AWS Provider Crossplane Blueprint Examples](../../../README.md)
- - [This serverless application](../object-processor-app/README.md)
+ - [Option 1: Container of this Go application](../object-processor-app/README.md)
```

Comment:
It mentions "Option 1:" where is option 2 in this doc?

Reply:
It is in the application readme: https://github.com/candonov/crossplane-on-eks/tree/change-lambda-to-container/examples/upbound-aws-provider/composite-resources/serverless-examples/object-processor-app#option-2-zip-file-to-s3